tutorials/xr/xr_room_scale.rst

```diff
@@ -68,6 +68,9 @@ We use this to determine where our body center is.
 
 Processing our character movement is now done in three steps.
 
+.. note:
```

Comment:
```suggestion
.. note::
```
Same with all, see the page itself, the first node does not show up, this is a comment instead

Reply:
For some reason I always get that one wrong.. sigh